src/pudl/helpers.py

```diff
@@ -1599,3 +1599,27 @@ def post_process(self, value: str) -> str:
                     f"Config value could not be found. Set the {self.env_var} environment variable or specify a value in dagster config."
                 )
         return value
+
+
+def get_asset_keys(
+    assets: list[AssetsDefinition], exclude_source_assets: bool = True
+) -> set[AssetKey]:
+    """Get a set of asset keys from a list of asset definitions.
+
+    Args:
+        assets: list of asset definitions.
+        exclude_source_assets: exclude SourceAssets in the returned list.
+                               Some selection operations don't require
```

Comment:
Do they not require `SourceAsset` keys, or not allow them?

Reply:
Ah good catch. I updated the doc string.